actionmailbox/test/dummy/db/migrate/20190731033524_add_service_name_column_to_active_storage_blobs.rb

```diff
@@ -0,0 +1,9 @@
+class AddServiceNameColumnToActiveStorageBlobs < ActiveRecord::Migration[6.0]
+  def up
+    return if column_exists?(:active_storage_blobs, :service_name)
+
+    if table_exists?(:active_storage_blobs)
+      add_column :active_storage_blobs, :service_name, :string
+    end
+  end
+end
```

Comment:
We shouldn't copy this migration to dummy apps. Instead, we should complement `activestorage/db/migrate/20170806125915_create_active_storage_tables.rb` migration by adding `t.string   :service_name`.
Also, we will need to apply the same change to `*_create_active_storage_table` migrations in the dummy apps. (I hope we'll remove dummy apps in favor of using railties tests soon.)

Reply:
Fixed, thanks!